### ISSUE TITLE
Unable to add account to Authy

### ISSUE THREAD

**Issue by myuseraccount** [scintill/DuOTP-android#5]
Setup fails for me, using Brave browser and Authy as my authenticator app.  I get this error from Authy:

invalid format
token could not be read

**Comment by liluzibird** [comment]
I am getting the same issue on andOTP, Aegis, and FreeOTP. 
Some processes I've done:
1. Scan QR code using the 2FA apps stated above
2. I pressed send as email and the duo://###... was sent to the browser, however I could not open it in DuOTP nor could I open them directly open them in any of my 2FA apps
3. This method, I scanned with my QR Analyzer app and tried to share to DuOTP app and hopefully get the secret token that way to manually enter to the app.

All of these methods have failed. The first one said the code isn't recognized, the second method, I failed to find the DuOTP app to share to. I looked through the list of Apps in the android settings and it was already installed property.


Edit:
I was able to open the duo://####... link with DuOTP app only after downloading the official Duo app through Aurora Store, however, it still won't do anything like redirecting to my 2fa app or provide me the secret token.
